astroquery/svo_fps/core.py

```diff
@@ -62,7 +62,7 @@ def data_from_svo(self, query, cache=True, timeout=None,
 
     def get_filter_index(self, wavelength_eff_min=0*u.angstrom,
                          wavelength_eff_max=FLOAT_MAX*u.angstrom, **kwargs):
-        """Get master list (index) of all filters at SVO
+        """Get main list (index) of all filters at SVO
```

Comment:
I thought it doesn't hurt but I'll revert as requested.

```suggestion
        """Get master list (index) of all filters at SVO
```